db/scripts/002_add_google_calendar.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE `qsignup`.`qsignups_regions`
+ADD COLUMN `google_calendar_id` VARCHAR(45) NULL AFTER `weekly_ao_reminders`;
```

Comment:
As I mentioned in slack, there could be a desire to have `google_calendar_id` at the AO level... for example, our website has individual pages for each AO, and before we moved to QSignups we would have a calendar specific to that AO on each page. Maybe that's something to consider in a V2 of this.

Reply:
Probably worth hopping on a call to talk about what *is* an AO.  As you've gotten requests from other regions, are we clear about what that represents?  If so - answers like this (calendar IDs) may fall in to place.